modules/userId/index.js

```diff
@@ -820,6 +821,16 @@ export function init(config) {
       auctionDelay = utils.isNumber(userSync.auctionDelay) ? userSync.auctionDelay : NO_AUCTION_DELAY;
       updateSubmodules();
     }
+    // userSync.ppid should be one of the 'source' values in getUserIdsAsEids() eg pubcid.org or id5-sync.com
```

Comment:
Hello, how/where would the publisher define the value for `userSync.ppid`?
Also, my understanding from this comment is that it will leave the choice to the publisher onto which ID from EIDS they choose to use as PPID: do you confirm?

Reply:
@FlorentDancy 

The pub will need to set `ppid` inside of their `userSync` setConfig.

Something like so:

```
      config.setConfig({
        userSync: {
          ppid: 'pubcid.org',
          userIds: [
            .. userIds
          ]
        }
      });
```

where the pubcid.org matches the userId submodules associated eids source value defined in `eids.js`

This PR has been closed as another one was open to fix the tests etc.

Reply:
Very clear, thanks @robertrmartinez!